docs/environment.yml

```diff
@@ -13,8 +13,8 @@ dependencies:
 - wheel=0.29.0=py27_0
 - zlib=1.2.8=3
 - pip:
-    - git+git://github.com/ga4gh/ga4gh-client.git@master#egg=ga4gh_client
-    - git+git://github.com/david4096/schemas.git@fixpip#egg=ga4gh_schemas
+    - ga4gh_common==0.0.5
+    - ga4gh_schemas==0.6.0a9
```

Comment:
We should be on post2 schemas around the board I think https://pypi.python.org/pypi/ga4gh-schemas/0.6.0a9.post2